### PR TITLE
Noora yaqub44/issue22

### LINKED ISSUE #22: Create the comment section card
**Is your feature request related to a problem? Please describe.**

**Describe how the solution should behave**
This will display the user comment, and it should also be able to render children as a sub comment. Recieve the info and child as props.

**Screenshots of the design/if any**
image

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by NooraYaqub44**
This is the Comment component but for now, I put the file inside pages so it would be easy to see and also used another background color so it could be easy to see and I will fix these two things after I get approvment.

### CODE REVIEW

File: pages/Comment.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  Button,
+} from "@chakra-ui/react";
+import { TiHeartOutline, TiArrowBackOutline } from "react-icons/ti";
+
+function Comment() {
+  const sampleComment = {
+    name: "Dan Abrahmov",
+    src: "https://bit.ly/dan-abramov",
+    content: "That is great",
+    likes: "56",
```
Comment (MuhannedNoman) [inline pages/Comment.js]: Let's have those passed to the component so we can use it, later on, like so. Also, let's add the logic to handle replay to comment functionality. The structure can be something like this [post ](https://stackoverflow.com/a/65010510)or [video ](https://www.youtube.com/watch?v=Hxgbiunp2DA)are doing

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
Hey, @NooraYaqub44 looks good, added a few comments to make things better. Let me know what you think.

**Review by aryankarim — APPROVED** [review]
Fix the conflicts, and implement Muhanned's note. Let me know if you need any help. Otherwise, this looks good.

**Review by MuhannedNoman — APPROVED** [review]
LGTM, feel free to merge.

**Pull request merged**